src/pages/Collection/Collection.jsx

```diff
@@ -1,40 +1,103 @@
-import React from "react";
+import React, { useEffect } from "react";
 import Card from "../../containers/Card";
-import { useSelector } from "react-redux";
+import { useSelector, useDispatch } from "react-redux";
+import PageTitle from "../../components/PageTitle";
 import {
   CollectionContainer,
   MockImage,
   EmptyCollectionContainer,
   EmptyTitle,
 } from "./styled";
 import noElementsImage from "../../assets/images/svg/father-and-daughter.svg";
-import { collectionSelector } from "../../store/ItemsCollection/selectors";
+import {
+  collectionSelector,
+  CollectionItemStatusSelector,
+  CollectionItemErrorSelector,
+} from "../../store/ItemsCollection/selectors";
+import { allCollectionsSelector } from "../../store/Collections/selectors";
+import { fetchItems } from "../../store/ItemsCollection/reducer";
+import { fetchCollections } from "../../store/Collections/reducer";
+import Loader from "../../components/Loader";
 
 function Collection() {
+  const collectionItemFetchStatus = useSelector(CollectionItemStatusSelector);
+  const allCollections = useSelector(allCollectionsSelector);
+  const collectionItemFetchError = useSelector(CollectionItemErrorSelector);
   const collectionItems = useSelector(collectionSelector);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchItems());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchCollections());
+  }, [dispatch]);
```

Comment:
у этих юзэффектов одна и та же зависимость, т. е и выполнятся они будут  одно и то же время -> мы можем объединить их в один

```suggestion
  useEffect(() => {
    dispatch(fetchItems());
    dispatch(fetchCollections());
  }, [dispatch]);
```